src/rdf4cpp/rdf/Literal.hpp

```diff
@@ -997,6 +1096,61 @@ class Literal : public Node {
      */
     [[nodiscard]] Literal sha512(NodeStorage &node_storage = NodeStorage::default_instance()) const;
 
+    /**
+     * returns the current time.
+     * Note: will need to be buffered for each query, because each query has only one now.
```

Comment:
Not sure what this comment means.

Reply:
I think what's meant here is that: 
```SPARQL
SELECT ?x ?y
WHERE {
  # Bind ?x and ?y to the current time
  BIND(NOW() AS ?x)
  BIND(NOW() AS ?y)
}
```

should return the same value for ?x and ?y. If we would call now() twice ?x and ?y will be different.

Reply:
Ok, but thats kinda obvious, right? Apart from that it's not even `rdf4cpp`'s responsibilty to handle that, so I find it confusing that it is even mentioned here.

Reply:
It's obvious if you think about it. But it's also that kind of thing one might overlook and that causes unnecessary bugs. So, I see no harm in leaving the comment there but would also add the example from my comment above to make it more self-contained.